install/kubernetes/cilium/templates/spire/agent/daemonset.yaml

```diff
@@ -0,0 +1,84 @@
+{{- if and .Values.auth.mTLS.spire.enabled .Values.auth.mTLS.spire.install.enabled -}}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: spire-agent
+  namespace: {{ .Values.auth.mTLS.spire.install.namespace }}
+  {{- with .Values.auth.mTLS.spire.install.server.annotations }}
+  annotations:
+    {{- toYaml . | nindent 8 }}
+  {{- end }}
+  labels:
+    app: spire-agent
+  {{- with .Values.auth.mTLS.spire.install.server.labels }}
+    {{- toYaml . | nindent 8 }}
+  {{- end }}
+spec:
+  selector:
+    matchLabels:
+      app: spire-agent
+  template:
+    metadata:
+      namespace: {{ .Values.auth.mTLS.spire.install.namespace }}
+      labels:
+        app: spire-agent
+    spec:
+      hostPID: true
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      serviceAccountName: {{ .Values.auth.mTLS.spire.install.agent.serviceAccount.name }}
+      initContainers:
+        - name: init
+          # This is a small image with wait-for-it, choose whatever image
+          # you prefer that waits for a service to be up. This image is built
+          # from https://github.com/lqhl/wait-for-it
+          image: cgr.dev/chainguard/wait-for-it@sha256:ecb58e3a2ffbdb732bb9049987e06eaf826d945410e167f31d6ffe28fab259f4
```

Comment:
`choose whatever image` sure but it is not a variable? 
If we pin on wait-for-it we should remove this comment

Reply:
Also comment seems lost, have we confirmed it runs on arm64 and amd64v1?